source/camera/basic/capture_2d.py

```diff
@@ -1,20 +1,42 @@
-"""Capture sample 2D."""
+"""
+This example shows how to capture 2D images from the Zivid camera.
+"""
+
 import datetime
 import zivid
 
 
 def _main():
     app = zivid.Application()
+
+    print("Connecting to camera")
     camera = app.connect_camera()
 
+    print("Configuring 2D settings")
     settings_2d = zivid.Settings2D()
     settings_2d.acquisitions.append(zivid.Settings2D.Acquisition())
-    settings_2d.acquisitions[0].aperture = 2.83
-    settings_2d.acquisitions[0].exposure_time = datetime.timedelta(microseconds=10000)
+    settings_2d.acquisitions[0].exposure_time = datetime.timedelta(microseconds=30000)
+    settings_2d.acquisitions[0].aperture = 11.31
+    settings_2d.acquisitions[0].brightness = 1.80
+    settings_2d.acquisitions[0].gain = 2.0
+    settings_2d.processing.color.balance.red = 1.0
+    settings_2d.processing.color.balance.green = 1.0
+    settings_2d.processing.color.balance.blue = 1.0
```

Comment:
Was this on purpose? Extending how many settings you set.

Reply:
Also, 11.31 is a really tight aperture, isn't it?

Reply:
I guess you are compensating with exposure time.

Reply:
Yes, to make it equivalent to C++ and C#